Command »npm start --port 8080« fails

The npm install and build steps work fine. However starting the server results in this:
```
jan@Rechenknecht:~/shout$ npm start -- --port 8080
npm ERR! Error: ENOENT, open '/home/jan/shout/node_modules/--port/package.json'
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! System Linux 3.19.0-31-generic
npm ERR! command "/usr/bin/node" "/usr/bin/npm" "start" "--" "--port" "8080"
npm ERR! cwd /home/jan/shout
npm ERR! node -v v0.10.28
npm ERR! npm -v 1.4.9
npm ERR! path /home/jan/shout/node_modules/--port/package.json
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /home/jan/shout/npm-debug.log
npm ERR! not ok code 0
```

Any idea?

--- a/README.md
+++ b/README.md
@@ -35,7 +35,7 @@ To run the app from source, just clone the code and run this in your terminal:
 ```
 npm install
 npm run build
-npm start -- --port 8080
+npm start
 ```
 
 ## License

